apps/sasquatch/src/projectDependency/java/com/microsoft/appcenter/sasquatch/activities/StorageActivity.java

```diff
@@ -42,16 +47,17 @@
 
 public class StorageActivity extends AppCompatActivity {
 
-    private ListView mListView;
+    public static ListView mListView;
+
+    public static ArrayList<String> mUserDocumentList = new ArrayList<String>();
```

Comment:
public variables do not start with "m": https://source.android.com/setup/contribute/code-style#follow-field-naming-conventions